src/Http/Client.php

```diff
@@ -479,7 +488,9 @@ public function send(RequestInterface $request, array $options = []): Response
         }
 
         do {
+            $this->dispatchEvent('HttpClient.beforeSend', [$request, $options]);
             $response = $this->_sendRequest($request, $options);
+            $this->dispatchEvent('HttpClient.afterSend', [$request, $options, $response]);
```

Comment:
```suggestion
            $this->dispatchEvent('HttpClient.afterSend', [$request, $response, $options]);
```

I think we should follow the conventions elsewhere in the framework and have `$options` be the last parameter.